Tests.Template/changelist.md

```diff
@@ -8,9 +8,13 @@ When the templates are used to create or update tests in a DSC Resource the vers
 * First release including version number.
 
 ## integration_template.ps1
+### Version 1.1.0
```

Comment:
If go to the "View" page you can see it doesn't get formatted properly.
Probably empty lines in between are required